packages/cli/src/UserManagement/email/NodeMailer.ts

```diff
@@ -1,23 +1,32 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 import type { Transporter } from 'nodemailer';
 import { createTransport } from 'nodemailer';
+import type { IDataObject } from 'n8n-workflow';
```

Comment:
please don't `IDataObject`. it's not much better than `any`.
`nodemailer` has the correct types for this. Please use `SMTPConnection.Options` or `SMTPTransport.Options`.

Reply:
@netroy cheers, Will fix that in the morning.